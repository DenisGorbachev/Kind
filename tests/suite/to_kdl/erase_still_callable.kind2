FnA #A {
  (+ FnB 1)
}

#kdl_erase
FnB #B {
  2
}