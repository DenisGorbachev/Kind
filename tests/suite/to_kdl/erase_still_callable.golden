// FnA : _
fun (A) {
  (A) = (+ (B) #1)
}

